src/commissaire_service/service/__init__.py

```diff
@@ -30,6 +30,33 @@
 from kombu.mixins import ConsumerMixin
 
 
+def add_service_arguments(parser):
```

Comment:
nit: If you don't believe this needs to be unittested go ahead and remove it from coverage via ``# pragma: no cover``.